CHANGELOG.md

```diff
@@ -55,6 +55,11 @@
 - When `singularity registry login` is used to login to a public OCI registry,
   the password is not verified at login time.
 - Improved the clarity of `singularity key list` output.
+- Do not mount current working directory when its path in the container and on
+  the host contain symlinks to different locations.
+- Create current working directory in container when it doesn't exist, so that
+  it can be entered. If a container is run from inside `$HOME`, `--no-mount
```

Comment:
I find this changelog message a bit unclear. In particular, it reads almost as if the user has to *supply* the `--no-mount` flag, as opposed to it being effectively inferred...

Reply:
You must now specify `--no-mount home,cwd` instead of just `--no-mount home` to avoid mounting from `$HOME` if you run singularity from inside `$HOME`.